smart-contracts/contracts/mixins/MixinLockMetadata.sol

```diff
@@ -14,11 +16,25 @@ import '../UnlockUtils.sol';
 contract MixinLockMetadata is
   IERC721,
   Ownable,
-  UnlockUtils
+  MixinLockCore,
+  UnlockUtils,
+  MixinKeys
 {
-  /// A descriptive name for a collection of NFTs in this contract
+  /// A descriptive name for a collection of NFTs in this contract.Defaults to "Unlock-Protocol" but is settable by lock owner
   string private lockName;
 
+  /// An abbreviated name for NFTs in this contract. Defaults to "KEY" but is settable by lock owner
+  string private lockSymbol;
+
+  // the base Token URI for this Lock. If not set by lock owner, the global URI stored in Unlock is used.
+  string private baseTokenURI;
+
+  IUnlock unlock = IUnlock(unlockProtocol);
```

Comment:
What's `unlockProtocol` here?

Reply:
Originaly it was the address of Unlock...now it seems to be the address of Unlock cast as an `IUnlock`

Reply:
I don't mind modifying this, but trying to use `unlockProtocol` directly didn't work for me. I decided to get it working and then refactor as needed. Open to ideas here.

Reply:
Got it.  Was probably some unrelated error.  I just gave it a shot and it works fine.  See the branch `hardlydifficult-unlockProtocol`

Reply:
ok cool. switching now.